ext/php8/ddtrace.c

```diff
@@ -165,6 +165,13 @@ ZEND_BEGIN_ARG_INFO_EX(arginfo_dd_trace_close_span, 0, 0, 0)
 ZEND_ARG_INFO(0, finish_time)
 ZEND_END_ARG_INFO()
 
+ZEND_BEGIN_ARG_INFO_EX(arginfo_dd_trace_set_distributed_tracing_context, 0, 0, 2)
+ZEND_ARG_INFO(0, trace_id)
+ZEND_ARG_INFO(0, parent_id)
+ZEND_ARG_INFO(0, origin)
+ZEND_ARG_INFO(0, tags)
```

Comment:
Back in the day, `origin` did not exist as a propagated header. That makes me think that we are likely to see new values supported in distributed tracing contexts in the future. I think trace ID and parent ID make sense as arguments, but I wonder if `origin` should be combined with `tags` as a generic vehicle to support new values in the future without having to update the API.

Reply:
I guess, if we now freshly introduced `origin`, after having `tags`, it would've gone into tags. But currently it is separate and probably should be reflected in the API as such as well?